react/src/js/components/Consonant/CardsCarousel/CardsCarousel.jsx

```diff
@@ -189,7 +267,27 @@ function CardsCarousel({
     const totalResultsHtml = RenderTotalResults(showTotalResultsText, resQty);
 
     useEffect(() => {
-        responsiveLogic();
+        mobileLogic();
+
+        const carousels = document.querySelectorAll('.consonant-Container--carousel');
```

Comment:
Instead of doing a query, you may want to try using a ref or at least memoize the card list so you’re not searching the entire DOM on every time.